vcd_cli/vapp.py

```diff
@@ -524,3 +522,28 @@ def add_disk(ctx, name, vm_name, size, storage_profile):
         stdout(task, ctx)
     except Exception as e:
         stderr(e, ctx)
+
+
+@vapp.command(short_help='set active vApp')
+@click.pass_context
+@click.argument('name', metavar='<name>', required=True)
+def use(ctx, name):
```

Comment:
Now that 'use' command has been added to vApp, I think we should make vAppname parameter optional for all the existing commands in vcd-cli/vapp.py t keep it consistent with org and vdc commands? What do you think?

Reply:
Yes, I agree.